<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="py_lib_tests" tests="1" failures="0" errors="0">
    <testcase name="py_lib_tests" status="run" duration="0" time="0"></testcase>
      <system-out>
Generated test.log (if the file is not UTF-8, then this may be unreadable):
<![CDATA[exec ${PAGER:-/usr/bin/less} "$0" || exit 1
Executing tests from //:py_lib_tests
-----------------------------------------------------------------------------
Former sys.path: ['/bazel_build/tools', '/root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles', '/root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles/__main__', '/root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles/bazel_tools', '/root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles/cyg_core_driver', '/root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles/lynx_core', '/root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles/mix_module_base_driver', '/root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles/mixfw_rules', '/root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles/mixrpc', '/usr/lib/python39.zip', '/usr/lib/python3.9', '/usr/lib/python3.9/lib-dynload', '/usr/local/lib/python3.9/dist-packages', '/usr/lib/python3/dist-packages']
Updated sys.path: ['/bazel_build/tools', '/root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles/__main__', 'tmp', '/usr/lib/python39.zip', '/usr/lib/python3.9', '/usr/lib/python3.9/lib-dynload', '/usr/local/lib/python3.9/dist-packages', '/usr/lib/python3/dist-packages']
============================= test session starts ==============================
platform linux -- Python 3.9.10, pytest-6.2.5, py-1.10.0, pluggy-1.0.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles/__main__/ic/tests, configfile: ../../pytest_py_lib_tests.ini
plugins: cov-2.12.1, flake8-1.0.7
collecting ... collected 3 items

ic/tests/test_mcp4725.py::FLAKE8 PASSED                                  [ 33%]
ic/tests/test_mcp4725.py::test_output_volt_dc PASSED                     [ 66%]
ic/tests/test_mcp4725.py::test_fast_output_volt_dc PASSED                [100%]

- generated xml file: /root/.cache/bazel/_bazel_root/9d15df359e2104d8be5679da2cc92856/execroot/__main__/bazel-out/k8-fastbuild/bin/py_lib_tests.runfiles/__main__/junit_pytest.xml -
============================== 3 passed in 0.23s ===============================
Copy coverage data .coverage.719bdacb8387.2182.528307 for py_lib_test]]>
      </system-out>
    </testsuite>
</testsuites>
